packages/ui5-config/src/ui5config.ts

```diff
@@ -168,6 +176,17 @@ export class UI5Config {
         return this;
     }
 
+    public updateUI5Libs(): void {
```

Comment:
Sorry, but no, now you have FE specific code in an even more generic module.
Please add something like an `addLibs(...libs: string[])` function and then call it in the FE specific code with 'sap.fe.templates` as parameter.

Reply:
Thanks for the fix! One suggestion though: consider using `add` instead of `update` otherwise, a consumer could assume a replacement of libs.